frontend/src/common.ts

```diff
@@ -700,6 +700,18 @@ export interface Class {
     events: ProgramClassEvent[];
 }
 
+export interface ClassStats {
```

Comment:
You can update the `Program` interface to include information about the enrollments and completions.